Block/Adminhtml/System/Config/Button/VersionCheck.php

```diff
@@ -66,6 +65,56 @@ public function _getElementHtml(AbstractElement $element): string
      */
     public function getVersion(): string
     {
-        return 'v' . $this->configRepository->getExtensionVersion();
+        return $this->configRepository->getExtensionVersion();
+    }
+
+    public function getLatestVersion()
+    {
+        $curlVersion = $this->getCurlVersion();
+        $this->curl->addHeader('Accept', 'application/vnd.github+json');
+        $this->curl->addHeader('User-Agent', 'curl/'.$curlVersion);
+        $this->curl->setOption(CURLOPT_RETURNTRANSFER, true);
+        $this->curl->get('https://api.github.com/repos/TrueLayer/magento2/releases');
+        $responseStatus = $this->curl->getStatus();
+        if ($responseStatus !== 200) {
+            $this->logger->error('Cron failed', [
```

Comment:
This message could be more meaningful if there's a way to actually see that message 😅

Reply:
It is visible to the admin in the error log, when manually viewing it. I updated the messages.